crates/bevy_ecs/src/schedule/executor/mod.rs

```diff
@@ -59,6 +70,7 @@ pub struct SystemSchedule {
     pub(super) system_dependents: Vec<Vec<usize>>,
     pub(super) sets_with_conditions_of_systems: Vec<FixedBitSet>,
     pub(super) systems_in_sets_with_conditions: Vec<FixedBitSet>,
+    pub(super) systems_with_stepping: FixedBitSet,
```

Comment:
`stepping_enabled_systems` perhaps?

Reply:
I went with the horribly long `systems_with_stepping_enabled` here.

My reasoning is that it matches the existing members of `SystemSchedule`:
* `sets_with_conditions_of_systems`
* `systems_in_sets_with_conditions`

Pattern seemed to be `system` or `set`, along with what was special.